examples/integration_with_error.py

```diff
@@ -104,7 +93,7 @@
         )
     )
     N = ndshape(manual_bounds)["t2"]
-    df = data.get_ft_prop('t2','df')
+    df = manual_bounds.get_ft_prop('t2','df')
     logger.debug(
```

Comment:
if this returns None for you, make sure you've pulled in most recent master of pyspecdata

Reply:
yes this should now work as it does in table_of_integrals!